README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## Self Introduction: Stan Xie
+
+Hello, I am **Stan Xie**, and am pursuing a master degree of arts in public policy with certificate in research methods at Harris. Prior to joining Harris, I received a bachelor’s degree in economics from Peking University, where I discovered a strong interest in social science research. My research interests include development economics, behavioral economics and political economy. 
+
+Specifically, I mainly focus on these topics below:
+
+* Social image
+* Social interaction
+* Social norm
+* Trust, belief and attitudes
+
+Last summer, I was a **DRW** fellow working as a summer RA at The Energy Policy Institute at UChicago (*EPIC*). And here is the link where I summerized my work there.
+
+[Work as a summer RA](https://epic.uchicago.edu/people/stan-xie/)
+
+Lastly, I am happy to share my headshot with you, which I took one year ago. During this pandemic period, it is a pity that we cannot sit together in the classroom. Hope everything will be done very soon!
+
+![Stan Xie](headshot_stan.jpg)
```

Comment:
Since the file "headshot_stan.jpg" refers to a local path, other people would not be able to see the photo unless they download the jpg file. I find that https://github.com/stanxie28/hw01/blob/master/headshot_stan.jpg?raw=true is the address for the uploaded photo file.